client/shipping-label/views/history.js

```diff
@@ -0,0 +1,79 @@
+import React, { PropTypes } from 'react';
+import CalypsoI18n from 'i18n-calypso';
+import Gridicon from 'components/gridicon';
+import formatDate from 'lib/utils/format-date';
+import { translate as __ } from 'lib/mixins/i18n';
+import { sprintf } from 'sprintf-js';
+const LabelHistory = ( { labelsHistory } ) => {
+	if ( ! labelsHistory || ! labelsHistory.length ) {
+		document.getElementById( 'woocommerce-order-label-history' ).style.display = 'none';
+		return null;
+	}
+	document.getElementById( 'woocommerce-order-label-history' ).style.display = 'block';
+
+	const renderUserName = ( entry ) => {
+		if ( entry.user_name ) {
+			return <a href={ entry.user_url }>{ entry.user_name }</a>;
+		}
+
+		return <span>{ __( 'User' ) }</span>;
+	};
+
+	const renderHistoryEntry = ( entry, index ) => {
+		const entryMoment = CalypsoI18n.moment( entry.time ).fromNow();
```

Comment:
Are we using Calypso's `moment()` anywhere else? I'm not sure we should be here.. from what I understand, `i18n-calypso` exposes a `moment` instance with a locale already set, but if we aren't using `i18n-calypso` for all translations in the project - can't this be a mismatch?

Why not use `moment` directly?